data/sql/updates/pending_db_world/rev_1555017745656363300.sql

```diff
@@ -0,0 +1,83 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1555017745656363300');
+
+-- Item should only drop from Riverpaw Mongrel (123)
+DELETE FROM `reference_loot_template` WHERE `Item`=821;
+-- Item should only drop from Sandcrawler  (830)
+DELETE FROM `reference_loot_template` WHERE `Item`=2087;
+-- Item is from crafting and should never drop
+DELETE FROM `reference_loot_template` WHERE `Item`=2300;
+
+-- Add unique drops to appropriate NPCs
+DELETE FROM `creature_loot_template` WHERE `Item` IN (821, 2087);
+INSERT INTO `creature_loot_template` (`Entry`, `Item`, `Chance`) VALUES
+(123, 821, 2.5),
+(830, 2087, 2);
+
+-- Kobold Vermin
+DELETE FROM `creature_loot_template` WHERE `Entry`=6;
+INSERT INTO `creature_loot_template` (`Entry`,`Item`,`Reference`,`Chance`,`QuestRequired`,`LootMode`,`GroupId`,`MinCount`,`MaxCount`,`Comment`) VALUES
+(6, 755, 0, 50, 0, 1, 0, 1, 1, NULL), -- Melted Candle
+(6, 4536, 0, 20, 0, 1, 0, 1, 1, NULL), -- Shiny Red Apple
+(6, 159, 0, 10, 0, 1, 0, 1, 1, NULL), -- Refreshing Spring Water
+(6, 24073, 24073, 10, 0, 1, 0, 1, 1, NULL), -- Junk
+(6, 11111, 11111, 0.5, 0, 1, 0, 1, 1, NULL); -- Bags
+
+-- Kobold Worker
+DELETE FROM `creature_loot_template` WHERE `Entry`=257;
```

Comment:
Also for these where you use Entry

Reply:
I think the statements are fine as they are, easier to read and better documented if they are structured in blocks for each creature. I also do the same in my PRs.